### PR TITLE
Add header and more design

### LINKED ISSUE #8: Add header and more design

### PR DESCRIPTION
**Pull request by sigtheidiot**
Closes #8 

This PR add's a header, a favicon and some more css

### CODE REVIEW

File: src/components/atoms/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "../../styles/header.css";
+
+function Header() {
+  return (
+    <div id="headerContainer">
+      <h1>Spicy sitater fra Krydderhyllen</h1>
+      <button className="button submitt">Har noen sakt noe dumt?</button>
```
Comment (ingraso) [inline src/components/atoms/Header.tsx:8]:
```suggestion
      <button className="button submitt">Har noen sagt noe dumt?</button>
```

File: src/styles/header.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#Header {
```
Comment (ingraso) [inline src/styles/header.css:1]: Nice if the ids are consistent, either camelCase, PascalCase or kebab-case. I don't remember how the names are in the other files, but maybe look them up and follow the same convention? In general, we should decide a convention to follow.

File: src/App.tsx [hunk 3]
```diff
@@ -11,20 +13,25 @@ function App() {
...
-            quoted={quote.quoted}
-            author={quote.author}
-            context={quote.context}
-            timestamp={quote.timestamp}
-          />
-        );
-      })}
-    </div>
+    <>
+      <div id="Header">
```
Comment (ingraso) [inline src/App.tsx:17]: This div could be removed. In Header.tsx you wrap the header with a div with the id "headerContainer". It is no need for two divs with different ids, when one of them only wraps around the other. Therefore, I think this div should be removed, so only the <Header />-component is present in this file.

**Review by ingraso — CHANGES_REQUESTED** [review]
I think this looks really good, nice work! :clap: :D

File: src/styles/header.css (hunk 2 above)
Comment (sigtheidiot) [reply src/styles/header.css:1]: I looks like everyone except for this one is camelCase, so i think thats what we're going for 😅

**Review by ingraso — APPROVED** [review]
Super :tada:

**Pull request merged**